src/asciidoc/templates/document.html.erb

```diff
@@ -0,0 +1,273 @@
+          <ul class="menu-drop-down">
+            <li><a href="/release-notes.html">Release Notes</a></li>
+            <li><a href="/javadoc.html">Javadoc</a></li>
+            <li><a href="/runtime-dependencies.html">Runtime Dependencies</a></li>
+          </ul>
+        </li>
+        <li><a href="/support.html">Support</a></li>
+        <li><a href="#">Contribute <i class="fa fa-chevron-circle-down" aria-hidden="true"></i></a>
+          <ul class="menu-drop-down">
+            <li><a href="/guidelines.html">Guidelines</a></li>
+            <li><a href="/components.html">Components</a></li>
+            <li><a href="/javastyle.html">Style Guide</a></li>
+            <li><a href="https://analysis.apache.org/dashboard/index/org.apache.logging.log4j:log4j">Sonar</a></li>
+            <li><a href="https://github.com/apache/logging-log4j2">GitHub</a></li>
+            <li><a href="https://logging.apache.org/team-list.html">Team</a></li>
+          </ul>
+        </li>
+        <li><a href="/security.html">Security</a></li>
+      </ul>
+    </nav>
+
+    <nav class="secondary-menu" aria-label="ASF related navigation">
+      <ul>
+        <li>
+          <a href="#">Apache <i class="fa fa-chevron-circle-down" aria-hidden="true"></i></a>
+          <ul class="menu-drop-down">
+            <li><a target="_blank" href="https://www.apache.org/">Home</a></li>
+            <li><a target="_blank" href="https://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
+            <li><a target="_blank" href="https://www.apache.org/licenses/">License</a></li>
+            <li><a target="_blank" href="https://www.apache.org/foundation/thanks.html">Thanks</a></li>
+            <li><a target="_blank" href="https://www.apache.org/events/current-event.html">Current Events</a></li>
+            <li><a target="_blank" href="https://www.apache.org/security/">Security</a></li>
+            <li><a target="_blank" href="https://privacy.apache.org/policies/privacy-policy-public.html">Privacy</a></li>
+          </ul>
+        </li>
+        <li><a href="https://logging.apache.org">Logging</a></li>
+      </ul>
+    </nav>
+  </div>
+</div>
+
+<div class="content-container">
+  <%
+    if (attr? :toc) && (attr? 'toc-placement', 'auto') %>
+    <div class="table-of-content">
+
+      <div class="table-of-content-title" id="toctitle"><%= attr 'toc-title' %></div>
+      <%= converter.convert self, 'outline' %>
+    </div><%
+    end
+  %>
+
+  <div id="content">
+      <h1><%= doctitle %></h1>
+      <%= content %>
+  </div><%
+    unless !footnotes? || attr?(:nofootnotes) %>
+      <div id="footnotes">
+        <hr><%
+        footnotes.each do |fn| %>
+          <div class="footnote" id="_footnote_<%= fn.index %>">
+            <a href="#_footnoteref_<%= fn.index %>"><%= fn.index %></a>. <%= fn.text %>
+          </div><%
+        end %>
+      </div><%
+    end %>
+</div>
+<% unless nofooter %>
+    <div id="footer">
```

Comment:
As a user I don't really see the footer while navigating and it is yet another thing to maintain next to many, and maintainers tend to forget to sync. the two. In short, I am not objecting. But I would be in favor of removing it to keep things simple.

Reply:
Fine for me. I added them because they are some kind of standard. I left the CSS in so far, since it is not much
Fixed in 4975a573ecb9ee89cc718b6bf851825d0a90b2d7